scss/partials/_about.scss

```diff
@@ -141,7 +147,7 @@ div#wrap {
         max-width: 744px;
         margin: 16px auto 0px;
 
-        @include mobile(){
```

Comment:
I'm not sure I understand the name change , `mobile` -> `tablet`.  Doesn't this css support both?

Reply:
before it was only for mobile (<768px) now it's for tablet too (<~1200px)